app/src/main/res/layout/activity_my_books.xml

```diff
@@ -6,12 +6,29 @@
     android:layout_height="match_parent"
     tools:context=".MyBooksActivity">
 
-    <!--    TODO: this is probably not the right way to do this :)-->
-    <fragment android:name="com.example.bookwormadventuresdeluxe2.MyProfile"
-        android:id="@+id/mainfrag"
+    <!-- Frame Container  -->
+    <FrameLayout
+        android:id="@+id/frame_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"/>
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@+id/bottom_navbar"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-    <!--    TODO: add navbar-->
+    <!-- Navbar   -->
+    <com.google.android.material.bottomnavigation.BottomNavigationView
```

Comment:
Could we get a screenshot of this?

Reply:
Sure. 
![Navbar](https://user-images.githubusercontent.com/44795394/96476524-8ec58800-11f2-11eb-8a56-705c5abda1c5.png)
P.S. "<!--    TODO: add navbar-->" was added previously by another team member.

Reply:
OMG it looks so good!! nicely done!